src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -26,25 +28,52 @@
-                    {:network-name :optimism :short-name "opt"}]
-    :state         :default
-    :action        :none}])
+(defn parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (filter (fn [{:keys [path]}]
+                 (not (string/starts-with? path constants/path-eip1581))))
+       (map (fn [{:keys [colorId emoji name address]}]
+              {:account-props {:customization-color (if (not-empty colorId) (keyword colorId) :blue)
+                               :size                32
+                               :emoji               emoji
+                               :type                :default
+                               :name                name
+                               :address             address}
+               :networks      [{:network-name :ethereum :short-name "eth"}
+                               {:network-name :optimism :short-name "opt"}
+                               {:network-name :arbitrum :short-name "arb1"}]
+               :state         :default
+               :action        :none}))))
 
-(defn view
+(defn keypair
+  [item index _ {:keys [profile-picture compressed-key]}]
+  (let [main-account (first (:accounts item))
+        color        (keyword (:colorId main-account))
+        accounts     (parse-accounts (:accounts item))]
+    [quo/keypair
+     {:customization-color (if (not-empty (:colorId main-account)) color :blue)
+      :profile-picture     (when (zero? index) profile-picture)
+      :status-indicator    false
+      :type                (if (zero? index) :default-keypair :other)
+      :stored              :on-device
+      :on-options-press    #(js/alert "Options pressed")
+      :action              :selector
+      :blur?               false
+      :details             {:full-name (:name item)
+                            :address   (when (zero? index)
+                                         (utils/get-shortened-compressed-key compressed-key))}
+      :accounts            accounts
+      :default-selected?   (zero? index)
+      :container-style     {:margin-horizontal 20
+                            :margin-vertical   8}}]))
+(defn- view-internal
   []
-  (let [{:keys [public-key compressed-key
-                customization-color]} (rf/sub [:profile/profile])
-        [display-name _]              (rf/sub [:contacts/contact-two-names-by-identity public-key])
-        profile-with-image            (rf/sub [:profile/profile-with-image])
-        profile-picture               (profile.utils/photo profile-with-image)]
+  (let [{:keys [compressed-key customization-color]} (rf/sub [:profile/profile])
+        profile-with-image                           (rf/sub [:profile/profile-with-image])
+        keypairs                                     (rf/sub [:wallet/keypairs])
+        profile-picture                              (profile.utils/photo profile-with-image)]
+    (rn/use-effect #(rf/dispatch [:wallet/get-keypairs]))
```

Comment:
Let's move this event inside the `:wallet/initialize` event, WDYT. 
The user can view the keypairs from the settings.

Reply:
@smohamedjavid fixed!